docs/tutorial/fuses.md

```diff
@@ -0,0 +1,52 @@
+# Electron Fuses
+
+> Package time feature toggles
+
+## What are fuses?
+
+For a subset of Electron functionality it makes sense to disable certain features for an entire application.  For example, 99% of apps don't make use of `ELECTRON_RUN_AS_NODE`, these applications want to be able to ship a binary that is incapable of using that feature.  We also don't want Electron consumers building Electron from source as that is both a massive technical challenge and has a high cost of both time and money.
+
+Fuses are the solution to this problem, at a high level they are "magic bits" in the Electron binary that can be flipped when packaging your Electron app to enable / disable certain features / restrictions.  Because they are flipped at package time before you code sign your app the OS becomes responsible for ensuring those bits aren't flipped back via OS level code signing validation (Gatekeeper / App Locker).
+
+## How do I flip the fuses?
+
+### The easy way
+
+We've made a handy module `@electron/fuses` to make flipping these fuses easy.  Check out the README of that module for more details on usage and potential error cases.
+
+```js
+require('@electron/fuses').flipFuses(
+  // Path to electron
+  require('electron'),
+  // Fuses to flip
+  {
+    runAsNode: false
+  }
+)
+```
```

Comment:
would be good if this had a command-line interface too, e.g. `npx @electron/fuses path/to/binary --set-fuse run_as_node=1`

Reply:
Potentially, I'm not a fan of adding a `bin` entry point for every module we write because it just adds extra overhead and writing a simple wrapper script isn't that hard.  Can discuss on the `fuses` repo

Reply:
it'll be handy for testing and manually editing, I think. also makes it easier to integrate into people's CI flows if they're not already all javascripty.